bakery/src/tasks/upload-book.js

```diff
@@ -63,8 +67,23 @@ const task = (taskArgs) => {
           aws s3 cp "$book_dir/collection.toc.json" "$toc_s3_link_json"
           aws s3 cp "$book_dir/collection.toc.xhtml" "$toc_s3_link_xhtml"
 
-          echo "$toc_s3_link_json" > upload-book/toc-s3-link-json
-          echo "$toc_s3_link_xhtml" > upload-book/toc-s3-link-xhtml
+          json_url=${cloudfrontUrlWithPrefix}/contents/$book_uuid@$book_version.json
+          xhtml_url=${cloudfrontUrlWithPrefix}/contents/$book_uuid@$book_version.xhtml
```

Comment:
Are these used anywhere? Or perhaps just staging them for now in case we decide to add them to `content_urls` later?

Reply:
Oopsies. yeah, these aren't really necessary. If we need to find what these are later, that's what git history is for. I'll delete them.